tutorial/001-the-host/README.md

```diff
@@ -0,0 +1,79 @@
+# The libp2p 'host'
+For most applications, the host is the basic building block you'll need to get
+started. This guide will show how to construct and use a simple host.
+The host is an abstraction that manages services on top of a swarm. It provides
+a clean interface to connect to a service on a given remote peer.
+
+First, you'll need an ID, and a place to store that ID. To generate a 'test'
+ID, you can do the following:
+```go
+import (
+	testutil "github.com/libp2p/go-testutil"
+	pstore "github.com/libp2p/go-libp2p-peerstore"
+)
+
+// For toy applications, this is an easy way to get an identity
+ident, err := testutil.RandIdentity()
```

Comment:
I don't like using `testutil` here because not what you use in reality and it's not helpful that the tutorial does not show how you do it properly. We should point people to https://godoc.org/github.com/libp2p/go-libp2p-crypto#GenerateKeyPair which can generate proper keys and identitity, just as easily, rather than letting the user figure it out when they're not doing "toys" anymore (imho).